examples/morpheusvm/vm/vm.go

```diff
@@ -33,7 +33,7 @@ func init() {
 	errs.Add(
 		// When registering new actions, ALWAYS make sure to append at the end.
 		// Pass nil as second argument if manual marshalling isn't needed (if in doubt, you probably don't)
-		ActionParser.Register(&actions.Transfer{}, actions.UnmarshalTransfer),
+		ActionParser.Register(&actions.Transfer{}, nil),
```

Comment:
confused by the register function. It uses avalanchego's linearcodec, so where does the new abi package play into here? 

https://github.com/ava-labs/hypersdk/blob/a230d1bbd8c152931c26593eb479c912b75cdc68/codec/type_parser.go#L40

Reply:
It is responsible for defining structs so they can be transferred between languages. If all implementations were in Golang, we wouldn't need an ABI.